conf/st2.dev.conf

```diff
@@ -46,7 +46,7 @@ action_executions_ttl = 30
 trigger_instances_ttl = 30
 
 [syslog]
-host = localhost
+host = 127.0.0.1
```

Comment:
We can not be sure that other services will be listening on `127.0.0.1`, same as ours which we explicitly bind to this addres.